helevents/templates/rest_framework/event_list.html

```diff
@@ -422,6 +422,11 @@ <h4 id="event-editable-events">Only editable events</h4>
         <pre><code>event/?admin_user=true</code></pre>
         <p><a href="?admin_user=true" title="json">See the result</a></p>
 
+        <p><code>registration_admin_user</code> parameter displays all events whose registration the user can modify</p>
```

Comment:
Maybe a `h4` title for this as well could be nice, maybe. Your call! :shrug:

Reply:
Added heading in commit https://github.com/City-of-Helsinki/linkedevents/pull/768/commits/81d24f2874a916092c848e1236e86b597ec8e64c